web/src/components/profile/ProfileAddressLockedFields.test.tsx

```diff
@@ -68,6 +83,8 @@ describe("ProfileAddressLockedFields", () => {
     });
     renderComponent({ address });
 
+    expect(screen.getByTestId("profile-business-address-locked-tooltip")).toBeInTheDocument();
+    expect(screen.getByTestId("locked-profileAddressEmpty")).toBeInTheDocument();
```

Comment:
[dust]` locked-profileAddressNotProvided`is another option and aligns better with the copy in the `JSON` and ~YAML` files.